### PR TITLE
copy edits and update to install dialog

### PR DESCRIPTION
**Pull request by laurensvalk**
This makes several copy edits and a slightly bigger revision to to the install dialog:

This makes the introduction paragraph and closing paragraphs into specific instructions. 

image

original:

image

### CODE REVIEW

**Comment by laurensvalk** [comment]
The recovery has a similar update and additional details about which apps to use thereafter:

image


original:

image

**Comment by laurensvalk** [comment]
It would probably be better to make the numbers consecutive. We can make some efforts to make the number of steps consistent across hubs or we add some tricks to make the start steps correct.

File: src/firmware/bootloaderInstructions/translations/en.json [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
         "learnMore": "Learn more."
     },
-    "instruction": "To flash the firmware, the hub must be placed in bootloader mode. {startPoweredOff}, then follow the instructions below:",
-    "startPoweredOff": {
-        "default": "Start with the hub powered off",
-        "usb": "Start with the hub powered off and the USB cable disconnected",
-        "recovery": "Start with the hub powered off and the {lego} PoweredUp app opened to any 'Create' program"
+    "prepare": {
+        "start": "Preparation:",
+        "batteries": "Make sure the hub has fresh batteries.",
```
Comment (dlech) [inline src/firmware/bootloaderInstructions/translations/en.json:9]: I'm not sure that "fresh" batteries is universally understood as fully charge/new.

File: src/tour/Tour.tsx [hunk 2]
```diff
@@ -96,11 +96,14 @@ const DownloadAndRunStep = React.memo(function DownloadAndRunStep() {
...
     return (
-        <p>
-            {i18n.translate('steps.downloadAndRun.message', {
-                icon: <Icon icon="play" style={{ verticalAlign: 'text-top' }} />,
-            })}
-        </p>
+        <>
+            <p>
+                {i18n.translate('steps.downloadAndRun.message', {
+                    icon: <Icon icon="play" style={{ verticalAlign: 'text-top' }} />,
```
Comment (dlech) [inline src/tour/Tour.tsx:102]: It would be better to put style in a .scss file.

Comment (laurensvalk) [reply src/tour/Tour.tsx:102]: OK - it was this way already so I had left it unchanged.

**Comment by dlech** [comment]
> It would probably be better to make the numbers consecutive.

The `<ol>` element has a `start` attribute. So we would just need a variable to count each enabled step to set that number.

**Comment by laurensvalk** [comment]
> The <ol> element has a start attribute. So we would just need a variable to count each enabled step to set that number.

I started using that in https://github.com/pybricks/pybricks-code/blob/3f6317a118f6f6b32612e0f118138ed901d339fc/src/firmware/bootloaderInstructions/BootloaderInstructions.tsx#L226 but it got a bit messy. I'll try the variable approach.

Firefox has a simple CSS solution though Chrome doesn't support it:

image

File: src/tour/Tour.tsx (hunk 2 above)
Comment (dlech) [reply src/tour/Tour.tsx:102]: ha, ok, don't worry about it

**Comment by dlech** [comment]
Maybe a way to do it with `counter()` in css? https://developer.mozilla.org/en-US/docs/Web/CSS/counter

**Comment by laurensvalk** [comment]
> Maybe a way to do it with `counter()` in css? https://developer.mozilla.org/en-US/docs/Web/CSS/counter

Seems like `counter-reset: none` does not work, but everything else does, so we can achieve it another way indeed :+1:

**Comment by laurensvalk** [comment]
Updated with the comments so far.

**Comment by laurensvalk** [comment]
Updated with fixes for comments so far.

**Comment by laurensvalk** [comment]
Can be merged if satisfactory. I'll make a separate PR for the next round.

File: src/firmware/bootloaderInstructions/translations/en.json [hunk 3]
```diff
@@ -1,34 +1,42 @@
...
     "step": {
-        "disconnectIo": "Disconnect all motors and sensors from the I/O ports on the hub.",
+        "start": "Enter update mode (see video):",
+        "disconnectIo": "Disconnect any device from port A on the hub.",
         "holdButton": "Press and hold the {button} on the hub.",
         "connectUsb": "Connect the USB cable.",
-        "waitForLight": "Keep holding the {button} and wait for the {light} to start flashing {lightPattern}. This takes about 5 seconds.",
-        "waitAppConnect": "The app will automatically connect and start flashing the firmware.",
+        "waitForLight": "Wait for the {light} to start blinking {lightPattern}.",
+        "waitAppConnect": "The app will automatically connect and start restoring the firmware.",
```
[the remark this answers is not in the document]
Comment (dlech) [reply src/firmware/bootloaderInstructions/translations/en.json:32]: Ah, yes, I see that now.

File: src/firmware/restoreOfficialDialog/RestoreOfficialDialog.tsx [hunk 4]
```diff
@@ -47,18 +51,25 @@ const RestoreFirmwarePanel: React.VoidFunctionComponent = () => {
...
+                            app:
+                                hubType === Hub.Inventor
+                                    ? legoMindstormsRegisteredTrademark
+                                    : legoEducationSpikeRegisteredTrademark,
+                        })}{' '}
+                        {hubType !== Hub.Inventor
+                            ? i18n.translate(
+                                  'restoreFirmwarePanel.instruction2.updateAppVersion',
+                              )
+                            : ''}
```
[the remark this answers is not in the document]
Comment (laurensvalk) [reply src/firmware/restoreOfficialDialog/RestoreOfficialDialog.tsx:72]: This puts the V2/V3 note with the wrong hub.

**Comment by laurensvalk** [comment]
Updated.

**Comment by laurensvalk** [comment]
Added a few more commits. I think this completes the first pass.

File: src/firmware/restoreOfficialDialog/RestoreOfficialDialog.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (dlech) [reply src/firmware/restoreOfficialDialog/RestoreOfficialDialog.tsx:72]: OK, I got the logic wrong, but the principal should still be able to be applied.

**Comment by dlech** [comment]
Need to update the tests to match the button text rename.

**Comment by laurensvalk** [comment]
Fixed

**Comment by dlech** [comment]
Thanks!

**Pull request merged**